packages/bruno-app/src/utils/common/regex.js

```diff
@@ -1 +1 @@
-export const envVariableNameRegex = /^(?!\d)[\w-]*$/;
+export const envVariableNameRegex = /^(?!\d)[\w-.]*$/;
```

Comment:
I believe `.` needs to be escaped in order to be considered a literal `.`, otherwise, `.` refers to _any_ character.

Reply:
Within a character set/class (`[]`) the `.` is treated as a literal instead of the usual "match any character" 👍

(side note: I used both https://regexr.com/ & https://regex101.com/ while testing, and the former showed no issues with this regex, but the latter displayed an error for the `-` which can be solved by escaping it. Not sure why there's a difference between them since they should both be using the browser's JS regex engine)